app/components/nav_bar/user_component.html.erb

```diff
@@ -11,24 +11,33 @@
                 <%= inline_svg_tag "logo.svg", class: "h-10 w-auto", alt: "RailsDevs logo", title: "RailsDevs logo" %>
               <% end %>
             </div>
-
             <!-- Navigation links -->
             <div class="hidden md:block">
               <div class="ml-10 flex items-baseline space-x-4">
                 <% links.each do |link| %>
                   <%= helpers.link_to_active link.title, link.path, active_class: "bg-gray-900 text-white", class: "text-gray-300 hover:bg-gray-700 hover:text-white px-3 py-2 rounded-md text-sm font-medium" %>
                 <% end %>
-
                 <% if hired? %>
                   <%= link_to hired_path, class: "hidden md:block text-gray-300 hover:bg-gray-700 hover:text-white px-3 py-2 rounded-md text-sm font-medium" do %>
                     <span class="absolute -mt-2.5 -ml-6 z-10 uppercase text-indigo-400 tracking-tight font-bold text-xs -rotate-12"><%= t(".new") %></span>
                     <span><%= t(".found_a_job") %></span>
                   <% end %>
+                  <%# For businesses with developer profiles %>
+                  <% if business? %>
+                    <%= link_to new_forms_businesses_hire_path, class: "hidden md:block text-gray-300 hover:bg-gray-700 hover:text-white px-3 py-2 rounded-md text-sm font-medium" do %>
+                      <span class="absolute -mt-2.5 -ml-6 z-10 uppercase text-indigo-400 tracking-tight font-bold text-xs -rotate-12"><%= t(".new") %></span>
+                      <span><%= t(".hired_a_dev") %></span>
+                    <% end %>
+                  <% end %>
+                <% elsif business? %>
+                  <%= link_to new_forms_businesses_hire_path, class: "hidden md:block text-gray-300 hover:bg-gray-700 hover:text-white px-3 py-2 rounded-md text-sm font-medium" do %>
+                    <span class="absolute -mt-2.5 -ml-6 z-10 uppercase text-indigo-400 tracking-tight font-bold text-xs -rotate-12"><%= t(".new") %></span>
+                    <span><%= t(".hired_a_dev") %></span>
+                  <% end %>
```

Comment:
I think this logic can be simplified now.

We are showing both CTAs if a user has both a business and developer profile. So we should be able to check `if developer?` and then `if business?`. Removing the `elsif` should mean not needing the code for the button twice.

Same goes for the mobile portion down below.

Reply:
currently using `hired?` here:

https://github.com/joemasilotti/railsdevs.com/blob/22f7bd53fccd36d6e505bcd69f73a785bdbef446/app/components/nav_bar/user_component.html.erb#L20

Do you prefer `developer?` here instead?

```ruby
# components/nav_bar/user_component.rb

def hired?
  user.developer&.persisted?
end

def developer?
  user.developer&.persisted?
end
```

Reply:
simplified logic in [0381715](https://github.com/joemasilotti/railsdevs.com/pull/818/commits/03817158013c41b57ea3f1630366320806f3bcb7)

used `developer?` here but can revert to `hired?` if you prefer

Reply:
Ah, we need to use the logic in `hired?` because we don't want it to show up on `/developers/new`. Same goes for the business!

I'm down for renaming the method to `developer?` and using that throughout - same for `business?`.